dcat/rdf/schema.ttl

```diff
@@ -15,6 +15,42 @@
 @prefix skos: <http://www.w3.org/2004/02/skos/core#> .
 @prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
 
+<http://purl.org/dc/elements/1.1/title>
+  owl:equivalentProperty schema:name ;
+.
+dcterms:description
+  owl:equivalentProperty schema:description ;
+.
+dcterms:format
+  owl:equivalentProperty schema:encodingFormat ;
+.
+dcterms:identifier
+  owl:equivalentProperty schema:identifier ;
+.
+dcterms:issued
+  owl:equivalentProperty schema:datePublished ;
+.
+dcterms:language
+  owl:equivalentProperty schema:inLanguage ;
+.
+dcterms:license
+  owl:equivalentProperty schema:license ;
+.
+dcterms:modified
+  owl:equivalentProperty schema:dateModified ;
+.
+dcterms:publisher
+  owl:equivalentProperty schema:publisher ;
+.
+dcterms:spatial
+  owl:equivalentProperty schema:spatial ;
+  owl:equivalentProperty schema:spatialCoverage ;
+.
+dcterms:temporal
+  owl:equivalentProperty schema:datasetTimeInterval ;
+  owl:equivalentProperty schema:temporal ;
```

Comment:
[schema:temporal](http://schema.org/temporal) and [schema:datasetTimeInterval](http://schema.org/datasetTimeInterval) have both been superseded by [schema:temporalCoverage](http://schema.org/temporalCoverage), so I wonder if we should not include them in the mapping

Reply:
Yes - I noticed that. For now I included everything I found in the prior work. The idea was to have a starting point to work from for a possibly normative alignment.